Formula/ksh.rb

```diff
@@ -16,6 +16,8 @@ class Ksh < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "8a6f8d5a96eaf9d91da32c58e453d7aacc4f47acea57f6a2b3db7cc108bbcd1f"
   end
 
+  deprecate! date: "2020-05-01", because: "abandoned, unmaintained, no releases since 2020; consider ksh93 instead"
```

Comment:
Do this in a separate PR to avoid the CI errors.

Reply:
Will do.

Reply:
https://github.com/Homebrew/homebrew-core/pull/106033/commits/1403294b44fe0e87b946c142c51483c6b7598767